### DISCUSSION TITLE
Pivot table on NocoDB [Jun 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2445] [Discord Threads]
> 🙋‍♂️ **José Manuel Aguirre**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Not yet, its been requested before I think and is tracked in github  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/985952952709697557)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
